datalad/__init__.py

```diff
@@ -16,6 +16,11 @@
 from .config import ConfigManager
 cfg = ConfigManager()
 
+from datalad.support.sshconnector import SSHManager
```

Comment:
Well, the thing is to have one central instance for datalad. So commands (or other code, using datalad python API)  can import the instance, instead of just the class. Of course we could make the SSHManager a singleton, too.
The current implementation lets GitRepo and AnnexRepo request connections from the SSHManager (these connections may or may not exist already at this point). I think it's nice to not need to care for it within application/command - code, when you don't explicitly use ssh in them. But this means: Both of the repo classes need to import it one way or another anyway. Whether it is an actual singleton or not, we cannot avoid the import, I think.